stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/NotificationHandler.kt

```diff
@@ -72,4 +72,8 @@ public interface NotificationHandler {
      * Dismiss all notifications.
      */
     public fun dismissAllNotifications()
+
+    public fun onNotificationPermissionGranted() { /* no-op */ }
+
+    public fun onNotificationPermissionDenied() { /* no-op */ }
```

Comment:
What is expected to be done on those methods?
Should our customers to implement anything here?

Reply:
These callbacks are for the cases when customers want to do something on their end when SDK requests notification permission. For both cases when either `Granted` or `Denied`